application/src/main/resources/tb-edge.yml

```diff
@@ -494,6 +494,9 @@ spring:
     open-in-view: "${SPRING_JPA_OPEN_IN_VIEW:false}"
     hibernate:
       ddl-auto: "none"
+      naming:
```

Comment:
@ertugertshkulaku 
Could you please provide more details on these changes below? Why do you think they are necessary?

Reply:
I encountered an error in creating the database and after changing this configuration I solved it, but I understand that it works without this configuration.  I will make a new commit where I have the tb_edge.yml in the initial state.